FrenchFilter/sections/general_extensions.txt

```diff
@@ -4,7 +4,7 @@
 !
 ! https://github.com/AdguardTeam/AdguardFilters/issues/74895
 lowend.xyz,veopartidos.net#%#//scriptlet("prevent-window-open")
-lowend.xyz#%#//scriptlet("abort-on-property-read", "popurl")
+dtefrc.online,lowend.xyz#%#//scriptlet("abort-on-property-read", "popurl")

```

Comment:
Also add `telerium.digital` here.

Reply:
Not reproduced, but okay.

Reply:
Sometimes when `telerium.digital` is the primary source, it's reproduced.